gumby/modules/tribler_module.py

```diff
@@ -126,19 +257,21 @@ def setup_initial_bootstrap_seeder(self):
                 fp.write(bytearray(random.getrandbits(8) for _ in range(bootstrap_size * 1024 * 1024)))
 
     @experiment_callback
-    def start_bootstrap_download(self):
-        self.session.start_bootstrap_download()
+    def start_bootstrap_download(self):  # obsolete?
```

Comment:
**Suggestion:** I think we still have some settings related to the bootstrap download in Tribler but the feature seems to be disabled. Feel free to remove this logic from Gumby if it is obsolete in Tribler 👍 .

Reply:
I removed it!